cmd/root.go

```diff
@@ -163,7 +163,7 @@ $ fioctl completion fish > ~/.config/fish/completions/fioctl.fish
 `,
 	DisableFlagsInUseLine: true,
 	ValidArgs:             []string{"bash", "zsh", "fish", "powershell"},
-	Args:                  cobra.ExactValidArgs(1),
+	Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
```

Comment:
finally! feel free to move this into its own PR and we can merge asap

Reply:
https://github.com/foundriesio/fioctl/pull/195